team-telnyx/webrtc
# fix(ENGDESK-8565): Update typedocs

- Update typedocs to solve vulnerability issues.


### Notes
 - Before merge this PR we need to make sure that `webrtc-api-docs` still working on developerdocs.

## 📝 To Do


## ✋ Manual testing

1. Navigate into `packages/js`
2. Run `npm i` and `npm run docs`
3. Check if the docs were generated correctly.

## 🦊 Browser testing

### Desktop
